src/nvim/normal.c

```diff
@@ -6320,11 +6320,16 @@ static void nv_pcmark(cmdarg_T *cap)
   const bool old_KeyTyped = KeyTyped;       // getting file may reset it
 
   if (!checkclearopq(cap->oap)) {
-    if (cap->cmdchar == 'g')
+    if (cap->cmdchar == TAB && mod_mask == MOD_MASK_CTRL) {
```

Comment:
docstring for `nv_pcmark` should be updated to mention `CTRL-Tab`

Reply:
Fixed.